client/js/components/map/map/DpOlMapEditFeature.vue

```diff
@@ -202,13 +183,41 @@ export default {
       }
     },
 
+    addInteraction (event) {
+      const id = 'selected' + uuid()
+
+      if (this.selectedFeatureId.indexOf(id) === -1) {
+        this.selectedFeatureId.push(id)
+        if (hasOwnProp(event, 'element')) {
+          event.element.set('id', id)
+          this.disabled = false
+        }
+      }
+    },
+
+    clearAll () {
```

Comment:
this is only moved to order it alphabetically.
I didn't want to reorder everything to not make it impossible to see the changes.